src/ixdat/techniques/ms.py

```diff
@@ -1123,122 +1230,22 @@ def gas_flux_calibration_curve(
         p_inlet=None,
         tspan_bg=None,
         ax="new",
-        axes_measurement=None,
-        axes_measurement_raw=False,
```

Comment:
you dropped that in the new method, but I'm not sure why? I just added this option in one of the more recent PRs to solve issues. We agreed that in the future this should be done better, but I can't see that you added a better option. Did I miss something there?

Reply:
Got it. Sorry. Shouldn't have dropped. I couldn't remember or quickly tell what it was. Reasons for that include:
- The indentation in the docstring was off so I couldn't immediately tell how much was part of the description for this variable.
- The name was wierd since it is a bool and not a list of axes. I've now given it a name `remove_bg_on_axis_measurement` which I think better representes the type and what it does.
- it was in the native but not siq version of the function
- ax**e**s_measurement was confusing since it is a single axis

I think I've fixed all of this now, and the option is there again! :)